tests/integration/route_test.py

```diff
@@ -0,0 +1,253 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+import copy
+
+import pytest
+
+from libnmstate import netapplier
+from libnmstate import netinfo
+from libnmstate.error import NmstateNotImplementedError
+from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
+from libnmstate.schema import InterfaceType
+from libnmstate.schema import Route
+
+IPV4_ADDRESS1 = '192.0.2.251'
+
+IPV6_ADDRESS1 = '2001:db8:1::1'
+
+ETH1_INTERFACE_STATE = {
+    Interface.NAME: 'eth1',
+    Interface.STATE: InterfaceState.UP,
+    Interface.TYPE: InterfaceType.ETHERNET,
+    Interface.IPV4: {
+        'address': [
+            {
+                'ip': IPV4_ADDRESS1,
+                'prefix-length': 24
+            }
+        ],
+        'dhcp': False,
+        'enabled': True
+    },
+    Interface.IPV6: {
+        'address': [
+            {
+                'ip': IPV6_ADDRESS1,
+                'prefix-length': 64
+            }
+        ],
+        'dhcp': False,
+        'autoconf': False,
+        'enabled': True
+    }
+}
+
+
+def test_add_static_routes(eth1_up):
+    routes = _get_ipv4_routes() + _get_ipv6_routes()
```

Comment:
For me it would be easier to understand if the methods names would mention that they return test routes instead of getting the current system routes, for example: `_get_ipv6_test_route_state()`.

Reply:
Changed to `_get_ipv6_test_routes()`